kymatio/scattering1d/frontend/sklearn_frontend.py

```diff
@@ -0,0 +1,15 @@
+from kymatio.frontend.sklearn_frontend import ScatteringTransformerMixin
+
+from kymatio.numpy import Scattering1D as ScatteringNumPy1D
+
+
+# NOTE: Order in base classes matters here, since we want the sklearn-specific
```

Comment:
Alright, it'd be great if this was in a generic class(this comment isn't specific to 1D), but I'm ok with it this way for now! Future cleaning

Reply:
Eh I don't think there's a way to do that without metaclasses...

Reply:
Arg. Ok. Solved.